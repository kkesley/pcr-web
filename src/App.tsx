import React from 'react'
import { BrowserRouter, Switch, Route } from 'react-router-dom'
import RecipeDetail from './pages/RecipeDetail'
import RecipeList from './pages/RecipeList'

function App() {
  return (
    <div className="App">
      <BrowserRouter>
        <Switch>
          <Route path={['/', '/recipes']} exact>
            <RecipeList />
          </Route>
          <Route path="/recipes/:recipeId">
            <RecipeDetail />
          </Route>
        </Switch>
      </BrowserRouter>
    </div>
  )
}

export default App
