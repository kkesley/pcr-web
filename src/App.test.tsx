import React from 'react'
import shallow from 'react-test-renderer/shallow'
import App from './App'

it('renders router and routes', () => {
    const renderer = shallow.createRenderer()
    const component = renderer.render(<App />)
    expect(component).toMatchSnapshot()
})
