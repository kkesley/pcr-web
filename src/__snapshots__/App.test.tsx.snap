// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders router and routes 1`] = `
<div
  className="App"
>
  <BrowserRouter>
    <AppRouter />
  </BrowserRouter>
</div>
`;
